biogrid/apps/biogrid-mvp/src/app/pages/infopage/infopage.spec.tsx

```diff
@@ -0,0 +1,21 @@
+import { cleanup, getByText, render, wait } from '@testing-library/react';
+import React from 'react';
+import InfoPage from './infopage';
+
+describe('Info page', () => {
+  afterEach(() => {
+    delete global['fetch'];
+    cleanup();
+  });
+
+  it('should render successfully', async () => {
+    global['fetch'] = jest.fn().mockResolvedValueOnce({
```

Comment:
nit, have 'my message' as a const variable that is reused in this mockResolvedValueOnce and in the await in line 19